app/javascript/lib/scp-api-metrics.js

```diff
@@ -145,16 +145,18 @@ export function logScatterPlot(
   perfTimes
 ) {
   const props = {
-    'numPoints': scatter.numPoints, // How many cells are we plotting?
+    numPoints: scatter.numPoints, // How many cells are we plotting?
     genes,
-    'gene': scatter.gene,
-    'is3D': scatter.is3D,
+    gene: scatter.genes,
```

Comment:
This seems like breaking change in analytics.  Previously, `gene` was a string.  Now it's an array.  I can't think of any analytics reports that currently use `gene`, but this will prevent analyzing that property it before our launch of this feature.  That said, I don't think it's a big deal, and I'd have done the same.

Two trivial change requests: 
* Make this `genes` like `getLogPlotProps`
* Add `numGenes` since it's trivial to implement, infeasible to derive in Mixpanel, and foreseeable we could support scatter plots with more genes

```suggestion
    genes: scatter.genes,
    numGenes: scatter.genes.length,
```

Reply:
technically, this isn't a breaking change -- this is a fixing change. :-)  The renaming of gene => genes happened in an unrelated earlier, already merged PR -- scatter.gene is always undefined now.  We're already storing a 'genes' param (line 149 above), so we can't call this 'genes' too.   Maybe call this one 'receivedGenes' -- maybe rename them both to receivedGenes and requestedGenes now since we haven't used either in a report yet?

Reply:
Ah, thanks for pointing out that `genes` is already logged.  I slightly prefer to keep property names as close as possible across interfaces (so e.g. `genes` instead of `requestedGenes`), but I don't object if you think it's worthwhile.

For easier visual grouping -- Mixpanel sorts properties alphabetically -- let's order property name parts as <subject><modifier> (e.g. `genesRequested`).  We use this order in e.g. #1056, with `numPointsPlotted` instead of `numPlottedPoints`.

I'd also opt for `genesPlotted` (and `numGenesPlotted`) over `genesReceived`, as it seems more descriptive.